editoast/src/views/v2/timetable.rs

```diff
@@ -219,6 +224,44 @@ async fn delete(
     Ok(HttpResponse::NoContent().finish())
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+#[schema(as=ConflictV2)]
+pub struct Conflict {
+    pub train_ids: Vec<i64>,
+    pub train_names: Vec<String>,
+    pub start_time: f64,
+    pub end_time: f64,
+    pub conflict_type: ConflictType,
+}
+
+/// Retrieve the list of conflict of the timetable (invalid trains are ignored)
+#[utoipa::path(
+    tag = "timetablev2",
+    params(
+        ("infra_id" = i64, Query, description = "The infra id"),
+    ),
+    responses(
+        (status = 200, description = "list of conflict", body = Vec<ConflictV2>),
+    ),
+)]
+#[get("/conflicts")]
+pub async fn conflicts(
+    _db_pool: Data<DbPool>,
+    _redis_client: Data<RedisClient>,
+    _core_client: Data<CoreClient>,
+    _infra_id: Query<i64>,
+) -> Result<Json<Vec<Conflict>>> {
+    // TODO
+    // issue: https://github.com/osrd-project/osrd/issues/6854
+    Ok(Json(vec![Conflict {
+        train_ids: vec![],
+        train_names: vec![],
+        start_time: 0.0,
+        end_time: 0.0,
```

Comment:
We should provide fake data like: 

```suggestion
        train_ids: vec![0, 1],
        train_names: vec!["train.1".into(), "train.2".into()],
        start_time: DateTime::from_timestamp(1710496800, 0),
        end_time: DateTime::from_timestamp(1710497400, 0),
```